mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -117,7 +133,7 @@ impl Bridge<WakingMemoryStore> {
 
 impl<S> Bridge<S>
 where
-    S: StreamWakeableState + Send,
+    S: StreamWakeableState + Send + Sync,
```

Comment:
I have gone ahead and commented out the in-memory solution for storage and have moved to ringbuffer which can be send and sync